packages/web/src/components/withdraw-usdc-modal/WithdrawUSDCModal.tsx

```diff
@@ -12,12 +21,53 @@ import { TransferInProgress } from './components/TransferInProgress'
 import { TransferSuccessful } from './components/TransferSuccessful'
 
 const messages = {
-  title: 'Withdraw Funds'
+  title: 'Withdraw Funds',
+  errors: {
+    insufficientBalance:
+      'Your USDC wallet does not have enough funds to cover this transaction.',
+    invalidAddress: 'A valid Solana USDC wallet address is required.',
+    pleaseConfirm:
+      'Please confirm you have reviewed the details and accept responsibility for any errors resulting in lost funds.'
+  }
+}
+
+export const AMOUNT = 'amount'
+export const ADDRESS = 'address'
+export const CONFIRM = 'confirm'
+
+const WithdrawUSDCFormSchema = (userBalance: number) => {
```

Comment:
whoa cool. can probably delete the slice/saga fns that validate the input now.

Reply:
doesn't hurt to do in both places, but i think this is the canonical way of doing input validation now! (learned this yday)